### ISSUE TITLE
Allow filtering of CWs

### ISSUE THREAD

**Issue by TiamatKefir** [glitch-soc/mastodon#2398]
### Pitch

Allow post filtering based on text in CW.

Useful when posts with CWs don't tag with the appropriate filter tag.

Example:

CW: politics
"talk about politics without the word politics in the main post"

With this feature users can drop hide posts based on CWs.

### Motivation

I see a lot of posts using CW, but I can't hide these posts if the poster doesn't use the word in the main body of text.
This would allow filtering based on CWs along with normal post filtering.

**Comment by Plastikmensch** [comment]
All filters already apply to CW, toot and alt text.
 
I'm not sure why it doesn't work for you.
Do you mind providing more information?

**Comment by TiamatKefir** [comment]
It may be the web viewers/apps I use then.
When I explore other instances other than my home instance, the filters do not work.
I can only speak for Firefox, Safari and several iOS apps; toot! tooot , and icecubes all do not pull my home instances filters.

If it's an instance issue, is there a chance to implement cross instance filtering?

**Comment by Plastikmensch** [comment]
> When I explore other instances other than my home instance, the filters do not work.
> I can only speak for Firefox, Safari and several iOS apps; toot! tooot , and icecubes all do not pull my home instances filters.

Yes, your filters only apply on your instance and are not shared with other instances.

> If it's an instance issue, is there a chance to implement cross instance filtering?

Not really. Instances operate independent of each other and implementing that would require to identify you to other instances and share user data, which is not desired.
